app/controllers/catalog_admin/item_types_controller.rb

```diff
@@ -58,7 +58,8 @@ def item_type_params
       :name_en, :name_plural_en,
       :name_fr, :name_plural_fr,
       :name_it, :name_plural_it,
-      :slug)
+      :slug,
+      :empty_fields)
```

Comment:
The name of the field is not very clear in my opinion. I would prefer it is something like `display_emtpy_fields`, or `show_empty_fields`.